### ISSUE TITLE
its possible check switch states?

### ISSUE THREAD

**Issue by Kbz4** [Napsty/check_shelly#2]
Hello, I think it is not possible but I want to make sure.
Is it possible to check the state of the output if it is On or Off?

I am trying to know it to know if it is active and there is no consumption, give the notice.
Thank you!

**Comment by Napsty** [comment]
Yes, should be possible. Good idea!

**Comment by wopfel** [comment]
I've added the check for the switch state / power state.

Using `--powerstate` you can specify the desired power state of the shelly switch.

I'll create a separate PR for this issue. I'm not very familiar with rebasing, so for my convenience I'd like to wait for the other PR to be closed / merged before creating a new PR 😊 

Here is a sample how it will look like:

```
# python3 /usr/local/bin/check_shelly.py -H 192.168.14.121 -t meter --powerstate 1 ; echo $?
SHELLY WARNING: Device (shellyplus1pm-xxx) SWITCH_0 is off, currently using 0 Watt / 0 Amp |power=0 current=0 total_power=50.983 temp=36.2 powerstatus=0
1

# python3 /usr/local/bin/check_shelly.py -H 192.168.14.121 -t meter --powerstate 0 ; echo $?
SHELLY OK: Device (shellyplus1pm-xxx) SWITCH_0 is off, currently using 0 Watt / 0 Amp |power=0 current=0 total_power=50.983 temp=36.0 powerstatus=0
0

# python3 /usr/local/bin/check_shelly.py -H 192.168.14.121 -t meter --powerstate 1 ; echo $?
SHELLY OK: Device (shellyplus1pm-xxx) SWITCH_0 is on, currently using 0 Watt / 0 Amp |power=0 current=0 total_power=50.983 temp=36.4 powerstatus=1
0

# python3 /usr/local/bin/check_shelly.py -H 192.168.14.121 -t meter --powerstate 0 ; echo $?
SHELLY WARNING: Device (shellyplus1pm-xxx) SWITCH_0 is on, currently using 0 Watt / 0 Amp |power=0 current=0 total_power=50.983 temp=36.2 powerstatus=1
1
```

**Comment by Napsty** [comment]
Looks good, thanks for these examples! 
And don't worry, that thing with code rebasing always hits me in the balls, too. I also prefer to do a proper new PR. :)

**Comment by Napsty** [comment]
Fixed in #5
